Cesium3DTilesSelection/test/TestTileLoadRequester.cpp

```diff
@@ -23,45 +23,51 @@ class TestTileLoadRequester : public TileLoadRequester {
     return !this->_workerThread.empty();
   }
 
-  Tile* getNextTileToLoadInWorkerThread() override {
+  const Tile* getNextTileToLoadInWorkerThread() override {
     CESIUM_ASSERT(!this->_workerThread.empty());
-    Tile* pResult = this->_workerThread.back();
+    const Tile* pResult = this->_workerThread.back();
     this->_workerThread.pop_back();
-    return pResult;
+    return const_cast<Tile*>(pResult);
   }
 
   bool hasMoreTilesToLoadInMainThread() const override {
     return !this->_mainThread.empty();
   }
 
-  Tile* getNextTileToLoadInMainThread() override {
+  const Tile* getNextTileToLoadInMainThread() override {
     CESIUM_ASSERT(!this->_mainThread.empty());
-    Tile* pResult = this->_mainThread.back();
+    const Tile* pResult = this->_mainThread.back();
     this->_mainThread.pop_back();
-    return pResult;
+    return const_cast<Tile*>(pResult);
```

Comment:
Doesn't this return a `const Tile*`? And `pResult` is already `const Tile*` ? 
```suggestion
    return pResult;
```

Reply:
Yep, that's left over from a previous iteration. Good catch!